spec/integrations/applicant_spec.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
 describe Onfido::Applicant do
-  subject(:applicant) { described_class.new }
```

Comment:
Since there's no global config, these classes can't really be constructed directly.